### PR TITLE
Fixes issue #2: Added  Material slider to adjust the brush size

### PR DESCRIPTION
**Pull request by Smarshal21**
Added  Material slider to adjust the brush size
Screenshot_1665141792
Screenshot_1665141835

https://user-images.githubusercontent.com/99678760/194550057-682bad55-c407-4e56-ab20-294204266e2f.mp4

### CODE REVIEW

**Comment by Smarshal21** [comment]
@rishidyno can you please review this PR

File: app/src/main/res/layout/dialog_brush_size.xml [hunk 1]
```diff
@@ -8,39 +8,19 @@
...
-            android:contentDescription="large image"
-            android:src="@drawable/ic_brush_size_large"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/brush_size_plus" />
-
 
     </androidx.constraintlayout.widget.ConstraintLayout>
 </layout>
```
Comment (rishidyno) [inline app/src/main/res/layout/dialog_brush_size.xml]: Add EOL

File: app/src/main/res/layout/dialog_brush_size.xml [hunk 2]
```diff
@@ -8,39 +8,19 @@
         android:gravity="center"
         android:orientation="vertical">
 
-        <ImageButton
-            android:id="@+id/brush_size_minus"
-            android:layout_width="wrap_content"
+        <com.google.android.material.slider.Slider
+            android:id="@+id/slidermo"
```
Comment (rishidyno) [inline app/src/main/res/layout/dialog_brush_size.xml]: Rename it to `brush_size_slider`

**Review by rishidyno — CHANGES_REQUESTED** [review]

**Comment by Smarshal21** [comment]
@rishidyno added the requested changes

**Comment by rishidyno** [comment]
LGTM

**Pull request merged**